feathr_project/feathr/utils/job_utils.py

```diff
@@ -38,22 +40,37 @@ def get_result_df(client: FeathrClient, format: str = None, res_url: str = None,
             delta = DeltaTable(local_dir_path)
             if not client.spark_runtime == 'azure_synapse':
                 # don't detect for synapse result with Delta as there's a problem with underlying system
-                # Issues are trached here: https://github.com/delta-io/delta-rs/issues/582
+                # Issues are tracked here: https://github.com/delta-io/delta-rs/issues/582
                 result_df = delta.to_pyarrow_table().to_pandas()
             else:
-                logger.info("Please use Azure Synapse to read the result in the Azure Synapse cluster. Reading local results is not supported for Azure Synapse. Emtpy DataFrame is returned.")
+                logger.info("Please use Azure Synapse to read the result in the Azure Synapse cluster. Reading local results is not supported for Azure Synapse. Empty DataFrame is returned.")
                 result_df = pd.DataFrame()
         elif format.casefold()=="avro":
             import pandavro as pdx
             for file in glob.glob(os.path.join(local_dir_path, '*.avro')):
                 dataframe_list.append(pdx.read_avro(file))
             result_df = pd.concat(dataframe_list, axis=0)
+        elif format.casefold()=="csv":
```

Comment:
>elif format.casefold()=="csv": [](http://example.com/codeflow?start=8&length=30)

Pls update get_result_df usage in doc to cover this

Reply:
updated in pydocs